python/ray/data/_internal/execution/operators/actor_pool_map_operator.py

```diff
@@ -117,7 +117,7 @@ def _dispatch_tasks(self):
             bundle = self._bundle_queue.popleft()
             input_blocks = [block for block, _ in bundle.blocks]
             ctx = TaskContext(task_idx=self._next_task_idx)
-            ref = actor.submit.options(num_returns="dynamic").remote(
+            ref = actor.submit.options(num_returns="dynamic", name=self.name).remote(
```

Comment:
@rkooo567 - I thought you mentioned there's problem that actor should not be set name here, right?

Reply:
So if we look at the dash screenshot above, I believe this isn't actually setting the name of the actor -- as we can see `_MapWorker` still being used in the `Active Actors by Name` chart. I think this part instead sets the name of the submitted task instead (e.g. `fn_actors` in above example).

Reply:
Also the name of actors should be unique across the cluster. So let's avoid setting the name for now.